Sprint-2/implement/lookup.test.js

```diff
@@ -33,3 +33,25 @@ It should return:
    'CA': 'CAD'
  }
 */
+test("should return the corresponding currency code", () => {
```

Comment:
The function does not actually return a "currency code".

Reply:
Thanks for your comment, I amended it.